### DISCUSSION TITLE
Showing audio output source on the tray icon

### DISCUSSION THREAD

**Discussion by es20490446e** [linuxdeepin/developer-center#3017] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hovering over the **audio tray** could show the current output source, so you don't have to visit the control center for figuring that out.

This is useful, for example, when you have **earphones** permanently connected to the computer.

**Comment by BLumia** [comment]
Probably related: https://github.com/linuxdeepin/developer-center/discussions/2926

**Comment by meiyixiang** [comment]
27/5000 
This is a question that we have looked at in the assessment how to achieve or at what stage

**Comment by meiyixiang** [comment]
Hello, we have accepted this requirement and will realize it later. You can pay attention to our update log or follow our version release to update
